src/loose/index.js

```diff
@@ -41,9 +41,7 @@ defaultOptions.tabSize = 4
 
 // eslint-disable-next-line camelcase
 export function parse_dammit(input, options) {
-  let p = new LooseParser(input, options)
-  p.next()
-  return p.parseTopLevel()
```

Comment:
Interesting that before it didn't pass a `node` to `parseTopLevel` but to align with the strict parse I went ahead and did that too. I also used `const` instead of `let` (though the `Parser` equiv should use `const` too).